epersongroups.md

```diff
@@ -0,0 +1,310 @@
+            "self": {
+              "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
+            }
+          },
+          "_embedded": {
+            "groups": {
+              "_embedded": {
+                "groups": []
+              },
+              "_links": {
+                "self": {
+                  "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+                }
+              },
+              "page": {
+                "number": 0,
+                "size": 0,
+                "totalPages": 1,
+                "totalElements": 0
+              }
+            }
+          }
+        }
+      ]
+    },
+    "_links": {
+      "self": {
+        "href": "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/groups"
+      }
+    },
+    "page": {
+      "number": 0,
+      "size": 10,
+      "totalPages": 1,
+      "totalElements": 10
+    }
+}
+```
+
+### Add a Group to a parent Group
+
+**POST /api/eperson/groups/<:parentgroupuuid>/groups**
+
+To add a child group to a parent group, perform a POST to the Sub groups of a Group endpoint when logged in as admin.
+
+The actual sub group is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X POST "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/05e3dbb8-332b-4487-a3f9-d78431b6cc02"
+```
+
+The group is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified group is not found
+
+### Remove a Group from a parent Group
+
+**DELETE /api/eperson/groups/<:parentgroupuuid>/groups**
+
+The actual sub group is part of the body using the uri-list
+
+Example:
+```bash
+curl -i -X DELETE "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+  -H "Content-Type:text/uri-list"
+  -d "https://dspace7.4science.it/dspace-spring-rest/api/eperson/groups/05e3dbb8-332b-4487-a3f9-d78431b6cc02"
+```
+
+The group is mandatory
+
+Return codes:
+* 204: if the update succeeded 
+* 401 Forbidden - if you are not authenticated
+* 403 Unauthorized - if you are not logged in with sufficient permissions 
+* 422: if the specified group is not found
+
+## Epeople in a single EPerson Group
+
+### Get Epeople for a single EPerson Group
+
+**GET /api/eperson/groups/<:uuid>/epersons**
+
+Filter using parameter:
+* all : retrieve indirect epeople as well
+
+Samples:
+* GET /rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons
+* GET /rest/api/eperson/groups/3b1de75d-5bf9-4ca3-bf4d-42bc8abd0d42/epersons?all=true
+
+
+```json
```

Comment:
better avoid examples here as it will be difficult to keep all of them up-to-dated